docker/Dockerfile-epoch

```diff
@@ -1,26 +1,23 @@
-FROM erlang:20.2 as builder
-
-ENV LIBSODIUM_VERSION=1.0.16
-RUN LIBSODIUM_DOWNLOAD_URL="https://github.com/jedisct1/libsodium/releases/download/${LIBSODIUM_VERSION}/libsodium-${LIBSODIUM_VERSION}.tar.gz" \
-    && curl -fsSL -o libsodium-src.tar.gz "$LIBSODIUM_DOWNLOAD_URL" \
-    && mkdir libsodium-src \
-    && tar -zxf libsodium-src.tar.gz -C libsodium-src --strip-components=1 \
-    && cd libsodium-src \
-    && ./configure && make -j$(nproc) && make install && ldconfig
+FROM aetrnty/builder as builder
```

Comment:
Do I understand correctly that this string here does not prevent user to build locally Dockerfile-builder then build this Dockerfile from that local builder? (i.e. this Dockerfile is not dependent on Docker Hub rather locally built builder takes the precedence.)

Reply:
If you create (build) the builder image and tag it `aetrnty/builder`, it would be used. Otherwise it will go to the hub and download the image named `aetrnty/builder`.